.changeset/five-donuts-enjoy.md

```diff
@@ -0,0 +1,7 @@
+---
+'@graphql-codegen/gql-tag-operations-preset': minor
+---
+
+The plugin now generates an `gql.ts` file for the `gql` function, whose contents are re-exported from the `index.ts` file. In module augmentation mode the `index.ts` file is omitted and only a `gql.d.ts` file is generated.
+
+Support for fragment masking via the new `fragmentMasking` configuration option. [Check out the Fragment Masking Documentation](https://envelop.dev/plugins/gql-tag-operations-preset#fragment-masking).
```

Comment:
@n1ru4l this link (https://www.envelop.dev/plugins/gql-tag-operations-preset#fragment-masking) is a 404 👀

Reply:
lol this should be graphql-code-generator